extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -418,10 +418,17 @@ class AmpLightbox extends AMP.BaseElement {
    * @private
    */
   maybeSetTransparentBody_() {
-    if (this.getAmpDoc().win != this.win) { // in FIE
-      setTransparentBody(this.getAmpDoc().win, /** @type {!HTMLBodyElement} */ (
-        dev().assert(this.win.document.body)));
+    if (!this.inFie_()) { // Not a FIE.
```

Comment:
use `getFriendlyIframeEmbedOptional` and check for null.

Reply:
(or better ad a `isFIE(iframe)` to the same file that has `getFriendlyIframeEmbedOptional`)

Reply:
There's no access to the frame from the lightbox element, unless we traverse up.